src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -262,6 +265,8 @@ const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ åpenBehand
                 <Alert variant="info">
                     {`Du er inne på en migreringsbehandling og det sendes ingen vedtaksbrev.`}
                 </Alert>
+            ) : erSmåbarnstilleggEndringFramITid ? (
+                <Alert variant="info">{`Du er inne på en behandling uten vedtaksbrev.`}</Alert>
             ) : (
                 <Alert variant="info">
                     {`Du er inne på en teknisk behandling og det finnes ingen vedtaksbrev.`}
```

Comment:
Kan vi forenkle disse uttrykkene litt? Hva med å feks ha `<boolsk uttrykk> && <komponent>` for hvert boolske uttrykk? Typ `erSmåbarnstilleggEndringFramITid && <Alert></Alert>`